bootstrap.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set +x
+
+# Ask the user for the installation directory
+read -p "Enter the installation directory (e.g., '.' or 'docs'): " install_directory
+
+# Clone the starter pack repository to a temporary directory
+echo "Cloning the starter pack repository..."
+git clone --depth 1 https://github.com/canonical/sphinx-docs-starter-pack temp-starter-pack
+rm -rf temp-starter-pack/.git
+
+# Update file contents for install directory
+echo "Updating working directory in workflow files..."
+sed -i "s|working-directory: .*|working-directory: $install_directory|g" "temp-starter-pack/.github/workflows"/*
```

Comment:
Sorry, I misread that as finding "." before. Another wildcard that is risky to replace imo.
Not sure if it's best to just match "." (and not replace anything if it was, for example, "./subdir"), or if we should replace the "." with "$install_directory".

Reply:
I just modified it. It now looks for `'.'` (including the quotes - as it appears in the starter pack today), and replaces with the install directory, also in quotes.